pkcs12/crypto.go

```diff
@@ -12,7 +12,7 @@ import (
 	"encoding/asn1"
 	"errors"
 
-	"golang.org/x/crypto/pkcs12/internal/rc2"
+	"github.com/keybase/go-crypto/pkcs12/internal/rc2"
```

Comment:
Change needed due to ["internal" package import restrictions](https://docs.google.com/document/d/1e8kOo3r51b2BWtTs_1uADIA5djfXhPT36s6eHVRIvaU)